### ISSUE TITLE
Add additional features to identify links in tables

### ISSUE THREAD

**Issue by bsaadat** [publichealthengland/coronavirus-dashboard#85]
Links in tables should be identified by more than just colour, e.g. by underlining them as this poses an accessibility issue.

**Comment by bsaadat** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add additional features to identify links in tables](https://trello.com/c/M1vZyzu6/189-add-additional-features-to-identify-links-in-tables)
